docs/source/guides/network_node/nucypher_host_management_cli.rst

```diff
@@ -0,0 +1,52 @@
+.. _managing-network-nodes:
```

Comment:
my preference would be to rename to something cloud-related - these references are global across the docs.

Same for the title on L4

Reply:
@derekpierre `managing-cloud-nodes`?
Please, this is more your area... I don't have an opinion.  I just want it to be coherent and make sense.

Reply:
`managing-cloud-workers`?
`managing-remote-staked-ursulas`?

Please make a suggestion and I'll commit it.

Reply:
+1 for *managing cloud workers from the CLI*

Reply:
+1 for `managing-cloud-workers`